src/test/find-users.test.ts

```diff
@@ -15,18 +15,91 @@ describe('Find Users query tests', () => {
     userRepository.delete({});
   });
 
-  it('Should return a list of Users', async () => {
-    const response = await getUsers(userPopulation);
+  it('Should return a list of All Users', async () => {
+    const response = await getUsers(1, userPopulation);
     const responseOutput = response.data.data.users;
 
     const usersFromDb = await getUsersFromDb();
 
-    expect(responseOutput).to.be.deep.eq(usersFromDb);
+    for (let i = 0; i < userPopulation; i++) {
+      expect(responseOutput.users[i].id).to.be.eq(usersFromDb[i].id);
+      expect(responseOutput.users[i].name).to.be.eq(usersFromDb[i].name);
+      expect(responseOutput.users[i].email).to.be.eq(usersFromDb[i].email);
+      expect(responseOutput.users[i].birthDate).to.be.eq(usersFromDb[i].birthDate);
+    }
+
+    expect(responseOutput).to.be.deep.eq({
+      total: userPopulation,
+      after: 0,
+      before: 0,
+      users: usersFromDb,
+    });
```

Comment:
Precisa desse for? Não estão sendo checados os campos do usuário no deep.eq de baixo?